src/main/java/examples/VertxMqttServerExamples.java

```diff
@@ -44,14 +44,21 @@ public void example1(Vertx vertx) {
     mqttServer.endpointHandler(endpoint -> {
 
       // shows main connect info
-      System.out.println("MQTT client [" + endpoint.clientIdentifier() + "] request to connect, clean session = " + endpoint.isCleanSession());
+      System.out.println(
+        "MQTT client [" + endpoint.clientIdentifier() + "] request to connect, clean session = "
+          + endpoint.isCleanSession());
 
       if (endpoint.auth() != null) {
-        System.out.println("[username = " + endpoint.auth().getUsername() + ", password = " + endpoint.auth().getPassword() + "]");
+        System.out.println(
+          "[username = " + endpoint.auth().getUsername() + ", password = " + endpoint.auth()
+            .getPassword() + "]");
       }
       if (endpoint.will() != null) {
-        System.out.println("[will topic = " + endpoint.will().getWillTopic() + " msg = " + new String(endpoint.will().getWillMessageBytes()) +
-          " QoS = " + endpoint.will().getWillQos() + " isRetain = " + endpoint.will().isWillRetain() + "]");
+        System.out.println(
+          "[will topic = " + endpoint.will().getWillTopic() + " msg = " + new String(
+            endpoint.will().getWillMessageBytes()) +
+            " QoS = " + endpoint.will().getWillQos() + " isRetain = " + endpoint.will()
+            .isWillRetain() + "]");
```

Comment:
why these changes?

Reply:
yes test scope would be fine too. Vert.x Core defines an optional dependency onto this artifact because it needs to compile with it.

Reply:
jetbrains idea format result. I will change it.